website/docs/d/vcenter.html.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_vcenter"
+sidebar_current: "docs-vcd-data-source-vcenter"
+description: |-
+  Provides a data source for vCenter server attached to VCD.
+---
+
+# vcd\_vcenter
+
+Provides a data source for vCenter server attached to VCD.
+
+Supported in provider *v3.0+*
```

Comment:
VCD version missing

Reply:
This works even with 9.7 so not sure we need it.